Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -17,7 +17,7 @@ UITestCase - a subclass wouldn't work with both of them.
 It's nicer code IMHO to have the state as a single associated property beacuse of the grossness of setting/getting it.
 This means that anytime I want to access my extra properties I just do `state.{propertyName}`
 */
-private class GherkinState {
+public class GherkinState {
```

Comment:
Why does this class have to become public - is it enough for it to become internal?